src/processes.c

```diff
@@ -2023,12 +2023,15 @@ static int ps_read (void)
 /* #endif HAVE_THREAD_INFO */
 
 #elif KERNEL_LINUX
-	int running  = 0;
-	int sleeping = 0;
-	int zombies  = 0;
-	int stopped  = 0;
-	int paging   = 0;
-	int blocked  = 0;
+	int running               = 0;
+	int sleeping              = 0;
+	int zombies               = 0;
+	int stopped               = 0;
+	int paging                = 0;
+	int blocked               = 0;
+	/* returns the number of processes with a command line */
```

Comment:
This doesn't "return" anything either. You could make this `/* number of processes with a command line */` and even move this to the end of the next line.
Also everywhere below.